src/fixtures/assets.js

```diff
@@ -54,6 +54,23 @@ export const initialAssets = [
     isDefault: true,
     balance: 0,
   },
+  {
+    address: 'mXx',
```

Comment:
What would we receive here?

Reply:
this is a sample fixture, the address is disregarded in the forming of the asset list.